pkg/jobstore/errors.go

```diff
@@ -6,154 +6,75 @@ import (
 	"github.com/bacalhau-project/bacalhau/pkg/models"
 )
 
-// ErrJobNotFound is returned when the job is not found
-type ErrJobNotFound struct {
-	JobID string
-}
-
-func NewErrJobNotFound(id string) ErrJobNotFound {
-	return ErrJobNotFound{JobID: id}
-}
-
-func (e ErrJobNotFound) Error() string {
-	return "job not found: " + e.JobID
-}
-
-// ErrJobAlreadyExists is returned when an job already exists
-type ErrJobAlreadyExists struct {
-	JobID string
-}
-
-func NewErrJobAlreadyExists(id string) ErrJobAlreadyExists {
-	return ErrJobAlreadyExists{JobID: id}
-}
+const (
+	JOB_STORE_COMPONENT = "JBS"
+	BOLTDB_COMPONENT    = "BDB"
+)
```

Comment:
What is the different between these components? `BOLTDB_COMPONENT` seems like an implementation detail.
Also, similar comment here regarding naming.